### PR TITLE
Trained new: Ethiopia_Tigray_Corrected_2020

### PR DESCRIPTION
**Pull request by bhyeh**
Retrained Ethiopia Tigray 2020 model w/ corrective label dataset.

### CODE REVIEW

File: data/models.json [hunk 1]
```diff
@@ -50,6 +50,23 @@
...
+            "precision_score": 0.6829,
+            "recall_score": 0.6043,
+            "roc_auc_score": 0.8407
+        },
+        "val_metrics": {
+            "accuracy": 0.8404,
+            "f1_score": 0.7261,
+            "precision_score": 0.7285,
+            "recall_score": 0.7237,
+            "roc_auc_score": 0.883
```
Comment (bhyeh) [inline data/models.json:67]: Bumped validation F1 metric from `0.6781` to `0.7261` but decrease in test F1.

**Comment by bhyeh** [comment]
Old model metrics
https://github.com/nasaharvest/crop-mask/blob/9b7f460eb1bbbeb32993a76e7cd583fcb36aded6/data/models.json#L21-L33
New model trained w/ corrective dataset metrics
https://github.com/nasaharvest/crop-mask/blob/9b7f460eb1bbbeb32993a76e7cd583fcb36aded6/data/models.json#L55-L67

**Comment by bhyeh** [comment]
@ivanzvonkov what was our consensus for moving forward here? retrain locally or await your removal of tracking normalizing dictionaries per model?

**Comment by ivanzvonkov** [comment]
This PR should fix the issue, though a retraining will be necessary to ensure that normalizing dictionary is saved: https://github.com/nasaharvest/crop-mask/pull/308

**Pull request closed without merging**